### ISSUE TITLE
[suggestion] use Medians instead of Averages

### ISSUE THREAD

**Issue by kentfredric** [Regner/albionmarket-frontend#5]
Averages are prone to value distortion by extreme outliers and so single individuals making stupid "1$" buy rates or "$1,000,000" sell rates can grossly move the "middle point"

Medians are less prone to this distortion as outliers only count for 1 additional unit.

It would probably be also useful to display Upper Quartile and Lower Quartile values of a given market.

That way the "good prices" are usually in between those two values, and getting lower than lower quartile is "really cheap" and higher than upper quartile is "really expensive".

**Comment by regner** [comment]
you are totally correct. This was a quick and dirty just get it done. Currently working on replacing the backend that gathers data. The new version will allow for tracking player locations. That should hopefully be done early next week. After that focus will come back to the actual market web site. This is a high priority change along with a history of some kind.
